src/github.js

```diff
@@ -68,24 +64,29 @@ const fetchFSTabCached = cache(fetchFSTabUncached, {
     owner,
     repo,
     ref,
-    options && options.headers && options.headers.Authorization
-      ? options.headers.Authorization : undefined,
+    options && options.headers && options.headers.authorization
+      ? options.headers.authorization : undefined,
```

Comment:
Maybe I'm mistaken, but isn't this the same as:
```
  options && options.headers && options.headers.authorization
```
At least I'm getting the same results...

Reply:
probably... this was like this before...but I'll make it nicer. I don't really the fact that there is always an undefined entry.